drivers/pulse_counter/pulse_counter.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2017 UC Berkeley
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/**
+ * @ingroup     drivers_pulse_counter
+ * @{
+ *
+ * @file
+ * @brief       Driver for the PULSE COUNTER.
+ *
+ * @author      Hyung-Sin Kim <hs.kim@cs.berkeley.edu>
+ *
+ * @}
+ */
+
+#include <string.h>
+
+#include "pulse_counter_params.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+/* Accumulate pulse count */
+static void pulse_counter_trigger(void* arg)
```

Comment:
parameter should be formatted like this: `void *arg`

Reply:
addressed